Jenkinsfile

```diff
@@ -18,7 +15,7 @@ properties([
     pipelineTriggers([pollSCM('H/15 * * * *')])
 ])
 
-node('docker') {
+node('sdk && docker') {
```

Comment:
We need some way to keep this build off arm64 nodes.  I already use the label `sdk` in the other  jenkins jobs https://github.com/coreos/jenkins-os/pull/1, so used it here.  Do you have an idea you think better?